FB.py

```diff
@@ -0,0 +1,106 @@
+import unittest
+from selenium import webdriver
+from selenium.webdriver.support.ui import Select
+import time
+from utils.constants import (FB_URL, FIRST_NAME, SUR_NAME, EMAIL_ADDRESS, NEW_PASSWORD)
+
+class SignUpForm():
+     "This class contains functions"
+
+     def find_element(self, css_selector='')
+
+         if css_selector:
+             return self.driver.find_element_by_css_selector(css_selector)
+
+     def clear_value_of_element(self, CSS_SELECTOR):
+
+         element = self.find_element(css_selector=CSS_SELECTOR)
+         element.clear()
+         return element
+
+     def enter_credentials_in_signUp_form(self, first_name='', last_name='',
+                                         email_address='', new_password=''):
+
+        selector_name = 'firstname'
+        selector_surname = 'lastname'
+        selector_email_id = 'u_0_r'
+        selector_password_id = 'u_0_w'
+        selector_reenter_id = 'u_0_u'
+        if first_name:
+        self.clear_value_of_element(CSS_SELECTOR=selector_name).send_keys(first_name)
```

Comment:
@ruttaba The indentation after the "if" condition is not correct. The line:
`self.clear_value_of_element(CSS_SELECTOR=selector_name).send_keys(first_name)` 
will always run.
Also fix this issue in other places.